### ISSUE TITLE
Elevation gain/loss not works

### ISSUE THREAD

**Issue by vsevjednom-cz** [OpenTracksApp/OpenTracks#756]
Your code is amazing but:

There is elevation gain/loss value during measure but without values (- m). I don't know why.

**To Reproduce**
1. Tap to "start".
2. Look at the elevation gain/loss. There is still only "- m".

**Technical information**
 - Device: Honor KSA-LX9
 - OS: Android
 - Version 9.1.0

**Comment by dennisguse** [comment]
I guess your telephone does not has a barometric sensor to measure pressure.
Using GPS altitude data is not (yet) implemented: #619

**Comment by vsevjednom-cz** [comment]
I understand. Is possible to add "Show/hide elevation gain and loss" to the settings?

**Comment by dennisguse** [comment]
@vsevjednom-cz At the moment, you  can only hide the altitude/elevation completely (absolute, gain/loss).

**Comment by vsevjednom-cz** [comment]
Yes, I know. But hide elevation gain/loss should be the solution for mobile phones without barometric sensor 🙃.

**Comment by dennisguse** [comment]
Yes, but no :)
Getting gain/loss from GPS values (with smoothing and some magic) is the actual solution aka #619.
I don't wont to spend time on a workaround. 
And the UI code is kind of complicated (less clean than it should be) - I am just reworking some part of the internal data transfer to avoid complexity.
It is a process....

TL;DR: If you want this now, go ahead.

**Comment by Sonegide** [comment]
Hi there,
I am having the same issue, with the elevation gain/loss not showing in the first tab.
The odd thing is that it is visible in the charts of the third and fourth tabs.
On older tracks, with the same phone, it was only showing the elevation gain, but not loss (while I always ended where I started).

**Comment by dennisguse** [comment]
@sonegide absolute elevation from the GPS is shown in the charts.
But these values cannot be directly used to get the elevation gain/loss.
We need to do some filtering etc before this is actual useful.
The issue is that GPS elevation is very noisy...

**Comment by Sonegide** [comment]
I don't see any absolute value in the chart (if I understand correctly, that you mean total value).
In case, I include a screenshot.

In any case I agree with the noise, therefore it may not be so relevant for me to get this total at this stage.
Anyway, thank you for this app, it's great  :)

![signal-2021-05-29-212522](https://user-images.githubusercontent.com/62804718/120082657-71c83000-c0c4-11eb-8fc8-0a59390f859a.png)
